File: WCF/Windows SDK WCF Samples/TechnologySamples/Scenario/DataBinding/WPFDataBinding/VB/service/IAlbumService.vb
' Copyright (c) Microsoft Corporation.  All Rights Reserved.

Imports System
Imports System.ServiceModel
Imports System.Collections
Imports System.Collections.Generic

Namespace Microsoft.ServiceModel.Samples

    <ServiceContractAttribute([Namespace]:="http://Microsoft.ServiceModel.Samples")> _
    Interface IAlbumService

        <OperationContract()> _
        Function GetAlbumList() As Album()
        <OperationContract()> _
        Sub AddAlbum(ByVal title As String)

    End Interface

End Namespace

File: WCF/Windows SDK WCF Samples/TechnologySamples Beta 1/Scenario/DataBinding/WPFDataBinding/CS/service/IAlbumService.cs

//  Copyright (c) Microsoft Corporation.  All Rights Reserved.

using System;
using System.ServiceModel;
using System.Collections;
using System.Collections.Generic;

namespace Microsoft.ServiceModel.Samples
{
	[ServiceContractAttribute(Namespace = "http://Microsoft.ServiceModel.Samples")]
    interface IAlbumService
    {
        [OperationContract]
        Album[] GetAlbumList();

		[OperationContract]
        void AddAlbum(string title);
    }
}
